lib/l10n_utils/__init__.py

```diff
@@ -147,9 +153,13 @@ def render(request, template, context=None, ftl_files=None, activation_files=Non
     context["template"] = template
     context["template_source_url"] = template_source_url(template)
 
+    # if it's a CMS page, draw the active locales from the Page data.
```

Comment:
For localised CMS pages, we're considering 'active' to be anything that has a translation and not (yet) considering how much of a version has actually been translated. That is on the list to add later